src/components/base/dropdown/NavigationDropdown.js

```diff
@@ -0,0 +1,167 @@
+import { encodeString } from "@/lib/utils";
+import Dropdown from "react-bootstrap/Dropdown";
+
+import Icon from "@/components/base/icon";
+import Text from "@/components/base/text";
+
+import Translate from "@/components/base/translate";
+import { useState, useEffect } from "react";
+import styles from "./NavigationDropdown.module.css";
+import classNames from "classnames/bind";
+import { useRouter } from "next/router";
+import Link from "../link/Link";
+
+export default function NavigationDropdown({ context, menuItems }) {
```

Comment:
Jeg har lige nogle spørgsmål angående den her. Jeg tror nemlig den kunne være noget mere fleksibel, men om vi gider at lave de ændringer afhænger af hvor meget arbejde det kræver.

Kan vi lige snakke sammen?

Reply:
Og lige som et recap af vores møde, så snakkede jeg om at bruge children i stedte for props og ift at bruge `compound components`. Men også at det er noget du kan gøre hvis det er praksisk at implementere og hvis du har lyst og tid